dvc/remote/base.py

```diff
@@ -253,7 +222,9 @@ def load_dir_cache(self, checksum):
             return []
 
         for info in d:
-            info[self.PARAM_RELPATH] = self.to_ospath(info[self.PARAM_RELPATH])
+            # NOTE: here is a BUG, see comment to .as_posix() below
```

Comment:
Can we put these notes in to_posix/from_posix instead of sprinkling bits everywhere? :slightly_smiling_face: Some of these will be deleted at some point and so the context will be lost for the others.

Reply:
There is nothing wrong with `.from_posix()` and `.as_posix()` we don't even have. The issue is how we use it in these particular places.

Reply:
This comments will go away once #2059 is fixed.

Reply:
@Suor but how would you find all of these comments that you've left? :) I don't think #2059 could be fixed, we could only forbid some chars, so you'll only need to modify from/to_posix, that would need to raise an error, so likely you won't get to removing these comments ever.

Reply:
For the record: agreed to merge this as is, but add links to these comments to #2059 so they don't get lost.